packages/core/src/index.ts

```diff
@@ -186,11 +169,11 @@ class Hopr extends EventEmitter {
       this.addressSorter = (x) => x
       log('Addresses are sorted by default')
     }
-    this.indexer = this.paymentChannels.indexer // TODO temporary
+    this.indexer = this.chain.indexer // TODO temporary
   }
 
   private async startedPaymentChannels(): Promise<HoprCoreEthereum> {
-    return await this.paymentChannels.start()
+    return await this.chain.start()
```

Comment:
`this.chain` seems to be missing as a class attribute?

Reply:
It’s defined in the constructor, making it a class property. Yeah, I also found it a bit confusing but that’s how Typescript does it.